docs/UserGuide.md

```diff
@@ -168,7 +173,8 @@ If no recurrence is set or there is no possible backlog, the output will be as f
 	Ok! The entry will not be backlogged
 	____________________________________________________________
 ```
-
+Note: If an entry of date beyond current date is added and user wishes to backlog, it will only add recurring entries until
+current date. <br>
```

Comment:
i thought backlogging only happens when the entry's date is before current date

Reply:
ya thats what i wrote, I think english issue, will fix

Reply:
done!